core/php/core.inc.php

```diff
@@ -46,18 +36,14 @@
 } catch (Error $e) {
 }
 
+/**
+ * Autoload function for specific Jeedom classes
+ * this function is called after the default Composer autoloader
+ * it will load specific classes such as Cmd, Real, etc.
+ * for plugins that do not use namespaces or Composer autoloading
+ */
 function jeedomAutoload($_classname) {
-	/* core class always in /core/class : */
-	$path = __DIR__ . "/../../core/class/$_classname.class.php";
-	if (file_exists($path)) {
-		include_file('core', $_classname, 'class');
-	} else if (substr($_classname, 0, 4) === 'com_') {
-		/* class com_$1 in /core/com/$1.com.php */
-		include_file('core', substr($_classname, 4), 'com');
-	} else if (substr($_classname, 0, 5) === 'repo_') {
-		/* class repo_$1 in /core/repo/$1.repo.php */
-		include_file('core', substr($_classname, 5), 'repo');
-	} else if (strpos($_classname, '\\') === false && strpos($_classname, '/') === false) {
+	if (strpos($_classname, '\\') === false && strpos($_classname, '/') === false) {
```

Comment:
(optionel) tu peux faire
```suggestion
	if (strpos($_classname, '\\') !== false && strpos($_classname, '/') !== false) {
	    return;
	}
```
et mettre le reste du code ensuite, ça réduira l’indentation, c’est toujours plus agréable à lire.